core/app/models/spree/order_updater.rb

```diff
@@ -163,6 +162,7 @@ def update_adjustment_total
       order.included_tax_total = all_items.sum(&:included_tax_total) + order_tax_adjustments.select(&:included?).sum(&:amount)
       order.additional_tax_total = all_items.sum(&:additional_tax_total) + order_tax_adjustments.reject(&:included?).sum(&:amount)
 
+      # TODO: Delete this line in Solidus 4.0, when it is run in `update_promotions`.
```

Comment:
@mamhoff while taking a look at this PR again, I notice this line. Is this comment still valid? I think we are already doing this in the `update_promotion` method so we can probably already remove this line?

Reply:
I diwscussed with @mamhoff about this, and it is actually valid. People might be calling `update_item_promotions` in their own store, which doesn't include this `promo_total` calculation.

Reply:
https://github.com/solidusio/solidus/pull/4890